### ISSUE TITLE
Launching from Terminal with path argument doesn't restore previously open windows

### ISSUE THREAD

**Issue by mcdado** [atom/atom#11368]
When launching from command line tool `atom` with a path argument while the app isn't open, any previously open window will not be restored.

**Reproduction Steps:**

1. Type in Terminal: `atom project1/`
2. Type in Terminal: `atom project2/`
3. Quit Atom app 
4. Type in Terminal: `atom project3/`

**Expected behavior:**

The windows with `project1` and `project2` should be restored, just like when typing just `atom` without argument at step 4.

**Observed behavior:**

After step 4, only `project3` window is displayed in Atom, `project1` and `project2` are discarded.

**Atom version:** Atom 1.6.2
**OS and version:** OS X 10.11.4

**Installed packages:**
```
Built-in Atom packages (89)
├── about@1.3.0
├── archive-view@0.61.0
├── atom-dark-syntax@0.27.0
├── atom-dark-ui@0.51.0
├── atom-light-syntax@0.28.0
├── atom-light-ui@0.43.0
├── autocomplete-atom-api@0.10.0
├── autocomplete-css@0.11.0
├── autocomplete-html@0.7.2
├── autocomplete-plus@2.25.0
├── autocomplete-snippets@1.10.0
├── autoflow@0.27.0
├── autosave@0.23.0
├── background-tips@0.26.0
├── base16-tomorrow-dark-theme@1.1.0
├── base16-tomorrow-light-theme@1.1.1
├── bookmarks@0.38.2
├── bracket-matcher@0.79.0
├── command-palette@0.38.0
├── deprecation-cop@0.54.0
├── dev-live-reload@0.47.0
├── encoding-selector@0.21.0
├── exception-reporting@0.37.0
├── find-and-replace@0.197.4
├── fuzzy-finder@0.94.0
├── git-diff@0.57.0
├── go-to-line@0.30.0
├── grammar-selector@0.48.0
├── image-view@0.56.0
├── incompatible-packages@0.25.0
├── keybinding-resolver@0.33.0
├── language-c@0.51.1
├── language-clojure@0.19.1
├── language-coffee-script@0.46.0
├── language-csharp@0.11.0
├── language-css@0.36.0
├── language-gfm@0.84.0
├── language-git@0.12.1
├── language-go@0.42.0
├── language-html@0.44.0
├── language-hyperlink@0.16.0
├── language-java@0.17.0
├── language-javascript@0.110.0
├── language-json@0.17.4
├── language-less@0.29.0
├── language-make@0.21.0
├── language-mustache@0.13.0
├── language-objective-c@0.15.1
├── language-perl@0.32.0
├── language-php@0.37.0
├── language-property-list@0.8.0
├── language-python@0.43.0
├── language-ruby@0.68.0
├── language-ruby-on-rails@0.25.0
├── language-sass@0.45.0
├── language-shellscript@0.21.0
├── language-source@0.9.0
├── language-sql@0.20.0
├── language-text@0.7.0
├── language-todo@0.27.0
├── language-toml@0.18.0
├── language-xml@0.34.2
├── language-yaml@0.25.1
├── line-ending-selector@0.3.0
├── link@0.31.0
├── markdown-preview@0.157.2
├── metrics@0.53.1
├── notifications@0.62.1
├── one-dark-syntax@1.2.0
├── one-dark-ui@1.1.9
├── one-light-syntax@1.2.0
├── one-light-ui@1.1.9
├── open-on-github@0.41.0
├── package-generator@0.41.0
├── settings-view@0.232.3
├── snippets@1.0.1
├── solarized-dark-syntax@1.0.0
├── solarized-light-syntax@1.0.0
├── spell-check@0.65.0
├── status-bar@0.83.0
├── styleguide@0.45.1
├── symbols-view@0.110.1
├── tabs@0.91.3
├── timecop@0.33.0
├── tree-view@0.203.2
├── update-package-dependencies@0.10.0
├── welcome@0.33.0
├── whitespace@0.32.1
└── wrap-guide@0.38.1

/Users/David/.atom/packages (40)
├── atom-autocomplete-php@0.18.7
├── atom-beautify@0.29.0
├── atom-html-preview@0.1.19
├── atom-terminal@0.8.0
├── autocomplete-html-entities@0.1.0
├── codebug@1.0.0
├── color-picker@2.1.1
├── dash@1.6.0
├── editorconfig@1.2.4
├── emmet@2.4.3
├── file-icons@1.7.0
├── html-entities@0.4.0
├── language-apache@1.4.0
├── language-ini@1.16.0
├── language-log@1.4.0
├── language-smarty@1.7.0
├── linter@1.11.4
├── linter-csslint@1.3.2
├── linter-htmlhint@1.1.1
├── linter-jscs@3.4.10
├── linter-jshint@2.0.4
├── linter-jsonlint@1.2.2
├── linter-php@1.2.0
├── linter-sass-lint@1.3.0
├── linter-scss-lint@2.5.1
├── linter-tidy@2.1.0
├── linter-xmllint@1.3.4
├── minimap@4.21.0
├── minimap-bookmarks@0.3.0
├── minimap-cursorline@0.1.0
├── minimap-find-and-replace@4.5.0
├── minimap-git-diff@4.2.0
├── minimap-linter@1.1.1
├── minimap-selection@4.4.0
├── native-ui@0.17.0
├── sort-lines@0.14.0
├── ssh-config@0.10.1
├── symbols-tree-view@0.11.0
├── synced-sidebar@0.4.3
└── w3c-validation@0.4.0
```

**Additional information:**

* Problem can be reproduced in safe mode: Yes
* Problem started happening recently, didn't happen in an older version of Atom: Don't Know
* Problem can be reliably reproduced, doesn't happen randomly: Yes
* Problem happens with all files and projects, not only some files or projects: Yes

**Comment by calebmeyer** [comment]
This is the behavior I expect. If I'm working on project1, then close atom and open it with a specific reference to project3, I don't want project1 to pop back up. If I want what I was working on previously, I'll just use `atom` with no arguments.

**Comment by mcdado** [comment]
@calebmeyer thanks for your reply. Personally, I'd like an option like that.
I'd prefer never to lose state, even the fact that when Atom restores the position of multiple windows they are all in the same position, using the coordinates of the top-most: it drives me crazy.

**Comment by mcdado** [comment]
For what it's worth, I found that this is a duplicate of: https://github.com/atom/atom/issues/10183

**Comment by damieng** [comment]
Closing this issue, track progress on the #10183
